<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电脑玩家聊天室</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser_info">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>361</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>njuszj</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>电脑玩家聊天室</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>v1.0测试版</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_input">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_login">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>登录/注册</string>
    </property>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>311</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>password</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>输入密码</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_usrname">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>输入用户名</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
